scripts/cc-cli/src/test/unit-tests/collect-coins.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  isExternalAddressValid,
+  isTxHashValid,
+} from "../../commands/collectCoins";
+
+describe("Test isTxHashValid", () => {
+  test("success", () => {
+    expect(
+      isTxHashValid(
+        "0x2446f1fd773fbb9f080e674b60c6a033c7ed7427b8b9413cf28a2a4a6da9b56c",
+      ),
+    ).toBeTruthy();
+  });
+
+  test("Fails with missing prefix", () => {
+    expect(
+      isTxHashValid(
+        "2446f1fd773fbb9f080e674b60c6a033c7ed7427b8b9413cf28a2a4a6da9b56c",
+      ),
+    ).toBeFalsy();
```

Comment:
Q: what's the actual return value here? I think tests could perform stricter assertions by asserting explictly on true/false values.

Reply:
Agreed. All of the methods tested in this PR should return explicit `true`/`false` values. Strict checks should be used.  I'll change the test assertations to be `expect(<fn call>).toBe(true | false)`